pkg/synthetictests/duplicated_events.go

```diff
@@ -21,7 +29,7 @@ func combinedRegexp(arr ...*regexp.Regexp) *regexp.Regexp {
 	return regexp.MustCompile(s)
 }
 
-var knownEvents = combinedRegexp(
+var allowedRepeatedEventPatterns = []*regexp.Regexp{
```

Comment:
allowedRepeatedEventPatterns take precedence over allowedRepeatedEventFns,  if an event match the former it will no be evaluated by the later, right?

Reply:
> allowedRepeatedEventPatterns take precedence over allowedRepeatedEventFns, if an event match the former it will no be evaluated by the later, right?

True.   Effectively that makes it a logical OR which I think people expect.